common/persistence/nosql/nosqlplugin/cassandra/visibility.go

```diff
@@ -80,6 +80,21 @@ const (
 		`AND start_time <= ? ` +
 		`AND workflow_id = ? `
 
+	// Intended for admin operations only
+	templateGetOpenWorkflowExecutionsByRunID = `SELECT ` + openExecutionsColumnsForSelect +
```

Comment:
with the quotes you should be able to do multiline without using the + for each line I should think. 

Given the `?` I take it this is a prepared statement?

Reply:
fair!